content/en/user-guide/integrations/crossplane/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Crossplane"
+tags: ["crossplane", "kubernetes", "infrastructure-as-code"]
+weight: 5
+description: >
+  Use the Crossplane cloud-native control plane framework with LocalStack
+aliases:
+  - /integrations/crossplane/
+---
+
+<img src="logo-crossplane.svg" width="500px" alt="Crossplane logo">
+
+## Overview
+
+[Crossplane](https://www.crossplane.io) is a cloud-native control plane framework, which offers an extensible backend that enables orchestrating applications and infrastructure via declarative APIs and resource definitions.
+
+Crossplane offers a native [AWS provider](https://github.com/upbound/provider-aws) which can be used to create and manage AWS cloud resources via the Crossplane platform.
+For example, it can be used to create S3 buckets, SQS queues, Lambda functions, among many other resources.
+At the time of writing, the Crossplane AWS provider supports a comprehensive set of some [900+ resource types](https://marketplace.upbound.io/providers/upbound/provider-aws).
+
+## Tutorial: Creating AWS resources in LocalStack via Crossplane
+
+In the following, we provide a step-by-step guide for installing Crossplane in a local test environment, and creating AWS resources (S3 bucket, SQS queue) in LocalStack via Crossplane.
+
+### Step 1: Installing Crossplane in local Kubernetes
+
+As a prerequisite, we need the following:
+* LocalStack running in local Docker
+* A local Kubernetes cluster:
+  * We can use the [embedded Kubernetes cluster](https://docs.docker.com/desktop/kubernetes) that ships with modern versions of Docker Desktop (can be easily enabled in the Docker settings)
+  * Alternatively, you can [create a local EKS cluster](https://docs.localstack.cloud/user-guide/aws/elastic-kubernetes-service/#create-an-embedded-kubernetes-cluster) in LocalStack directly, which will spin up a light-weight embedded `k3d` Kubernetes cluster in your Docker environment
+* The [`helm`](https://helm.sh) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) command-line clients installed
+
+Once your `kubectl` is configured to point to the local Kubernetes cluster, we first install Crossplane via `helm`:
+{{<command>}}
+$ helm repo add crossplane-stable https://charts.crossplane.io/stable
+$ helm repo update
+$ helm install crossplane crossplane-stable/crossplane --namespace crossplane-system --create-namespace
+{{</command>}}
+
+The installation may take a few minutes. In parallel, we can install the `crossplane` command-line extensions for `kubectl`:
+{{<command>}}
+$ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | bash
+...
+$ sudo mv kubectl-crossplane /usr/local/bin
+{{</command>}}
+
+To confirm that the installation was successful, we can run these `kubectl` commands, which should yield output similar to the following:
+{{<command>}}
+$ kubectl crossplane --version
+v1.13.2
```

Comment:
I think we want to remove output from command sections so copy and paste works better?

Reply:
Good point - see my comment in the PR description above. I'm sort of a fan of the terse syntax with commands&output in a single listing, as otherwise the length of the text kind of explodes. Will change it if it's the general consensus, but I'm not sure it makes for a great visual experience if we _always_ have a separate listing with the command, followed by some prose text like _"This command will yield the following output"_ , followed by another listing with the output text. 

What I would basically like to have is a `<command>` tag that has two properties:
1. special handling of `$` (not getting selected when user selects the code with the mouse)
2. **no** "Copy" button in the top right corner

If I use a standard code block with backticks, it doesn't satisfy point (1), and if I use a `<command>` tag, it doesn't satisfy point (2). 🤷 

Maybe @HarshCasper or @lukqw @webdev51 can help us out here..?

Reply:
Hi, I made 2 improvements to the `command.html` file which is responsible for rendering the `<command>` short code .

1. Added an option to pass a prop called `wrapper` which (as the name suggests) will simply wrap the `command` box in a wrapper `div` which won't do anything but the thing is, if you pass the wrapper called `wrapper="disable-copy-btn"`, we have some styles going on which will hide the copy button. So, it'll look like this:  `<command wrapper="disable-copy-btn">}} ... {{</command>` and we'll get a command with no copy button

2. I also added a wrapper tag called  `<disable-copy>` which works like this:
Let's say we've a command with some output:
`{{<command>}}
$ kubectl crossplane --version
<disable-copy>v1.13.2</disable-copy>
{{</command>}}`

The above command will still have a copy button but clicking "Copy"  will exclude that text inside the `disable-copy` tags

Please let me know if anything can be improved in this and can I push the changes in the same PR? Both the examples are there in my local branch for now. I can also create a new PR if that's better. 

CC: @lukqw , @whummer , @HarshCasper

Reply:
Hey @webdev51 , thanks for jumping on this so quickly! 🚀 

I think both options are great. 👌  I have a slight preference for `<disable-copy>`, as it combines the best of both worlds - allowing the copying of the command, while still allowing non-copyable output in the code listing. 

Please feel free to push your changes to the branch directly! 🙌

Reply:
@whummer, I did the changes with `<disable-copy>` but still left the `wrapper` functionality as well for future if we ever need to completely hide the "Copy" button.

Would be great if you just have a quick final look :smile: 

I'll also mention it in the #sig-docs maybe so everyone is aware of these changes